src/blocks/accordion-item/edit.js

```diff
@@ -8,22 +15,41 @@ import { __ } from '@wordpress/i18n';
  * @param {Object}   props               Block Props.
  * @param {Object}   props.attributes    Block Attributes.
  * @param {Function} props.setAttributes Set Block Attributes.
- * @param {string}   props.context       Block Context.
  * @see https://developer.wordpress.org/block-editor/reference-guides/block-api/block-edit-save/#edit
  *
  * @return {Element} Element to render.
  */
-export default function Edit( { attributes, setAttributes, context } ) {
-	const blockProps = useBlockProps(); // eslint-disable-line react-hooks/rules-of-hooks
-	const fontColor = context[ 'accordion/fontColor' ];
-
-	if ( fontColor !== attributes.fontColor ) {
-		setTimeout( () => setAttributes( { fontColor } ) );
-	}
+export default function Edit( { attributes, setAttributes } ) {
+	const { borderColor } = attributes;
+	const colorPalette = useSelect(
+		( select ) => select( 'core/block-editor' ).getSettings().colors || [],
+		[]
+	);
+	const blockProps = useBlockProps();
 
 	return (
 		<div { ...blockProps }>
-			<div className="wmf-accordion-item" aria-expanded>
+			<InspectorControls>
+				<Panel header={ __( 'Set border color:', 'wmf-reports' ) }>
+					<PanelBody>
+						{ borderColor && (
+							<p>{ __( 'Border Color', 'wmf-reports' ) }</p>
+						) }
+						<ColorPalette
+							value={ borderColor }
+							colors={ colorPalette }
+							onChange={ ( color ) =>
+								setAttributes( { borderColor: color } )
+							}
```

Comment:
What do you think about using the palette picker I introduced in #116? It would give this control more UI consistency with other blocks, and reduce a lot of this markup (would effectively replace the entire `<InspectorControls>` tree)

Reply:
Yes agree for both of these points, I went with this because it was already set up for this block. Will switch 👍